### PR TITLE
ci(github): move connect webextension example tests to GitHub Actions

### LINKED ISSUE #7916: Move Connect popup tests from GitLab to GitHub Actions
We want to move all Connect popup tests from GitLab to GitHub Actions.

We have started with that with this PR https://github.com/trezor/trezor-suite/pull/11304

We need some follow-ups:

- [x] Add schedule to connect tests in GitHub Actions
- [x] Migrate tests transport e2e
- [x] Migrate tests .e2e connect-web
- [x] Migrate tests  connect legacy fws
- [x] Migrate tests  connect canary fws
- [x] Migrate tests connect examples webextensions
- [x] Migrate tests connect T2B1
- [x] Deploy static link with test results like https://github.com/trezor/trezor-suite/blob/develop/docs/tests/e2e-connect-popup.md#test-results as commented here https://github.com/trezor/trezor-suite/pull/11304#discussion_r1520151634
- [x] Migrate tests connect-popup legacy npm package

Maybe we also want to create a GitHub workflow with all the Connect Tests, e2e and unit. This will allow us to have a single definition for the paths that we want to watch.

### PR DESCRIPTION
**Pull request by karliatto**
## Description
Migrate from GitLab to GitHub Actions example webextension test.

Move the tests for example webextension to popup tests since they are using the same infrastructure and it was confusing to have those in `connect-webextension` package because that package now is used for different reason.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/7916

### CODE REVIEW

**Comment by karliatto** [comment]
@SocketSecurity ignore

File: .github/workflows/template-connect-popup-test-params.yml [hunk 1]
```diff
@@ -92,14 +100,23 @@ jobs:
...
+        run: |
           yarn workspace @trezor/connect-webextension build
           yarn workspace @trezor/connect-iframe build:core-module
           yarn workspace @trezor/connect-explorer build:webextension
 
+      - name: Build example webextension
+        run: |
+          yarn workspace @trezor/connect-web build:webextension
+          yarn workspace @trezor/connect-web build:inline
+          node packages/connect-examples/update-webextensions.js
```
Comment (martykan) [inline .github/workflows/template-connect-popup-test-params.yml]: It's not called with parameters like before (`--trezor-connect-src "${URL}" --npm-src "${URL}/trezor-connect.js"`) is this intended?

Comment (karliatto) [reply .github/workflows/template-connect-popup-test-params.yml]: Yes, that was missed, I just added it. Thanks!

File: .github/workflows/template-connect-popup-test-params.yml [hunk 2]
```diff
@@ -92,16 +100,24 @@ jobs:
         run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
         id: extract_branch
 
-      - name: Build webextension
+      - name: Install dependencies
         run: |
           yarn install --immutable
           yarn build:libs
```
Comment (vdovhanych) [inline .github/workflows/template-connect-popup-test-params.yml]: we should use directly building just the necessary lib, not all of them, right?

**Review by vdovhanych — COMMENTED** [review]
Otherwise i think its fine

File: .github/workflows/template-connect-popup-test-params.yml (hunk 2 above)
Comment (mroz22) [reply .github/workflows/template-connect-popup-test-params.yml]: yes, that shouldn't be needed globally here.

File: .github/workflows/template-connect-popup-test-params.yml [hunk 3]
```diff
@@ -92,16 +100,24 @@ jobs:
...
 
-      - name: Make Docker Script Executable
-        run: chmod +x ./docker/docker-connect-popup-ci.sh
+      - name: Build example webextension
+        env:
+          URL: https://${{ inputs.DEV_SERVER_HOSTNAME }}/connect/${{ steps.extract_branch.outputs.branch }}/
+        run: |
+          yarn workspace @trezor/connect-web build:webextension
+          yarn workspace @trezor/connect-web build:inline
+          node packages/connect-examples/update-webextensions.js --trezor-connect-src "${URL}" --npm-src "${URL}trezor-connect.js"
```
Comment (mroz22) [inline .github/workflows/template-connect-popup-test-params.yml:118]: so url here is just regular branch of a PR right?  (or maybe develop branch)

also didn't we use it somewhere in combination production URLs?

Comment (karliatto) [reply .github/workflows/template-connect-popup-test-params.yml:118]: Yes, URL here is that, the branch of the PR where this test is running. On schedule it is develop.

File: .github/workflows/template-connect-popup-test-params.yml (hunk 2 above)
Comment (karliatto) [reply .github/workflows/template-connect-popup-test-params.yml]: You are right, that `yarn build:libs` was totally unnecessary, done  https://github.com/trezor/trezor-suite/pull/12011/commits/1c66aef5021ddcd26a75bc52924455b99025cb39

**Comment by karliatto** [comment]
so what do you think guys, it is ok alreay?  @vdovhanych @martykan @mroz22

**Review by vdovhanych — APPROVED** [review]

**Comment by karliatto** [comment]
@SocketSecurity ignore

**Review by vdovhanych — APPROVED** [review]

**Pull request merged**